src/models/autocompleteservice/AutocompleteResponse.ts

```diff
@@ -93,5 +93,9 @@ export interface AutocompleteResult {
    * Any vertical keys associated with a prompt.
    * This only shows up on universal autocomplete requests.
    **/
-  verticalKeys?: string[]
+  verticalKeys?: string[],
+  /**
+   * {@link SearchIntent}s corresponding to the autocomplete result.
+   */
+  queryIntents: SearchIntent[]
```

Comment:
What's the reason for naming this field `queryIntents`, when the similar field in `AutocompleteResponse` is called `inputIntents`?